compiler/modules/importer.nim

```diff
@@ -294,7 +294,7 @@ proc transformImportAs(c: PContext; n: PNode): tuple[node: PNode, importHidden:
     ret.node = n.processPragma
   return ret
 
-proc myImportModule(c: PContext, n: var PNode, importStmtResult: PNode): PSym =
+proc myImportModule(c: PContext, n: var PNode, info: TLineInfo, importStmtResult: PNode): PSym =
```

Comment:
```suggestion
proc myImportModule(c: PContext, n: var PNode, info: TLineInfo,
                    importStmtResult: PNode): PSym =
  ## `info` provides the source position (which may be different from the one
  ## of `n`) to use for symbol suggestions.
```
It's not immediately obvious why this procedure needs an extra `TLineInfo`, so documenting it makes sense.